front/public/locales/fr/simulation.json

```diff
@@ -20,6 +20,9 @@
   "isWorking": "Calcul de la simulation en cours…",
   "labels": "Étiquettes",
   "loadingTrain": "Chargement du train",
+  "manchetteSettings": {
+    "waypointsVisibility": "Modifier la visibilité des PRs"
+  },
```

Comment:
```suggestion
  "manchetteSettings": {
    "waypointsVisibility": "Afficher/masquer les PRs"
  },
```
**or above**

Reply:
`Modifier la visibilité des PRs` is the text on the sketch

Reply:
so this one seems better

Reply:
@thibautsailly wants to keep the `Modifer la vi...`